README.md

```diff
@@ -96,6 +96,19 @@ You might also like to check out `hermeto --help` and the `--help` texts of the
 
 ## Configuration
 
+### Modes
+
+Hermeto can be run in two modes using the global CLI option `--mode`.
+
+* strict
+* permissive
+
+The default mode is `strict`. In this mode, **some** input requirements that are not met are treated
+as errors. On the other hand, the `permissive` mode treats them as warnings. This means that Hermeto
+will proceed, but it may not be able to produce a complete and accurate SBOM.
```

Comment:
This is a bit unclear: it makes it look like it is possible to have an error ignoring which would still result in a complete and accurate SBOM. Did you mean something like "Hermeto will not fail and generate an SBOM, but it will contain omissions and inaccuracies of varying degree."?